libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1034,24 +1034,32 @@ void AP_GPS::Write_DataFlash_Log_Startup_messages()
 
 /*
   return the expected lag (in seconds) in the position and velocity readings from the gps
+  return true if the GPS hardware configuration is known or the delay parameter has been set
  */
-float AP_GPS::get_lag(uint8_t instance) const
+bool AP_GPS::get_lag(uint8_t instance, float &lag_sec) const
 {
     // return lag of blended GPS
     if (instance == GPS_MAX_RECEIVERS) {
-        return _blended_lag_sec;
+        lag_sec = _blended_lag_sec;
+        // auto switching uses all GPS receivers, so all must be configured
+        return all_configured();
```

Comment:
Technically not a problem now, but if we ever support 3 or more GPS units the blended instance could only be 2 of the units, and then it would only matter if the gps's that are contributing to blending have been configured not all of them. But there are a lot of other GPS units > 3 issues to be addressed then as well so we can wait to address it.